tests/test_orbit.py

```diff
@@ -4280,6 +4280,163 @@ def test_full_plotting():
     else: raise AssertionError("plot3d(d3='vy') applied to RZOrbit did not raise AttributeError")
     return None
 
+def test_from_name():
```

Comment:
Split these tests into a few more specific tests:
  * Basic test that the correct object is found
  * Tests of the various errors
  * Tests of the warnings
  * ...

This way it would be easier to tell if one of the tests fail which part of the code is the issue.